### ISSUE TITLE
Is this project continued?

### ISSUE THREAD

**Issue by jaekook-neonesia** [mxschmitt/playwright-go#48]
Few days ago, there was a playwright-go link at playwright official page(Supported language).

But now, it's gone.

Could I use this this lib?

**Comment by mxschmitt** [comment]
Hi, I would say it's not official but lead by an active contributor. We added it to the docs after the 1.4 release that's why it's only shown on - GitHub or master: https://playwright.dev/#version=master&path=docs%2Flanguages.md&q=

**Comment by jaekook-neonesia** [comment]
Thank you for fast reply and contribution.

I will use this lib.

**Comment by mxschmitt** [comment]
> Thank you for fast reply and contribution.
> 
> I will use this lib.

We are really curious for what are you using this lib. Is it for scraping, end-to-end tests or something else? Would help us really much to know more about our target group. Thanks!

**Comment by jaekook-neonesia** [comment]
> > Thank you for fast reply and contribution.
> > I will use this lib.
> 
> We are really curious for what are you using this lib. Is it for scraping, end-to-end tests or something else? Would help us really much to know more about our target group. Thanks!
It's for scraping. I used chromedp and rod. Those lib has problem with frame handling. 
So, I tested puppeteer and playwright. But I don't want to use node for service.
At that time, I found this lib official playwright home page.

I'm working at Indonesia(I'm Korean). In here, many old website use frameset and javascript session update. 
Headless scraping is need for job automation.

Thank you.
